gizmo.py

```diff
@@ -0,0 +1,97 @@
+# First draft for gizmo.py
+import numpy as np
+import pandas as pd
+
+def hello(name, country='Finland'):
+    print('Hello ' + name + ', how are things in ' + country + '?')
+
+def spell(word):
+
+    output = ""
+    for i,l in enumerate(word):
+        if i == len(word)-1:
+            output = output + l
+        else:
+            output = output + l + "."
+
+    print(output)
+
+def relative_path(ids):
+
+    list = []
+    path_base = './subjects/mock_recording_'
+    suffix = '.rec'
+    for id in ids:
+        list.append(path_base + id + suffix)
```

Comment:
Here, [f-strings](https://realpython.com/python-f-strings/) are a clear win:

```python
list.append(f'./subjects/mock_recordings_{id}.rec')
```

much more readable than needing separate variables for `path_base` and `suffix`.